### PR TITLE
Added EF cancellation to timeline

### PR DESCRIPTION
**Pull request by trevorm4**
Added EF cancellation to timeline, so that you can see which EF you cancelled in specific

Courtesy of Abe

### CODE REVIEW

**Comment by abelito75** [comment]
Thanks for making this change for me. I was a bit lazy to boot up the server and make a new branch since mine is currently flooded with fallen order memes

File: analysis/monkmistweaver/src/modules/spells/EssenceFont.tsx [hunk 1]
```diff
@@ -37,7 +37,8 @@ class EssenceFont extends Analyzer {
   cancelled_ef: number = 0;
   hasUpwelling: boolean = false;
   cancelDelta: number = 100;
-  last_ef: number = 0;
+  last_ef_time: number = 0;
+  last_ef: any = null;
```
Comment (abelito75) [inline analysis/monkmistweaver/src/modules/spells/EssenceFont.tsx]: BAD
NEVER DO ANY. @Pewtro WILL MURDER YOU IF YOU DO
You should do CastEvent | null

**Review by abelito75 — CHANGES_REQUESTED** [review]
ANY BAD

File: analysis/monkmistweaver/src/modules/spells/EssenceFont.tsx (hunk 1 above)
Comment (trevorm4) [reply analysis/monkmistweaver/src/modules/spells/EssenceFont.tsx]: Fixed

**Pull request merged**